demos/python_demos/smart_library_demo/README.md

```diff
@@ -0,0 +1,133 @@
+The following pretrained models can be used:
+
+* `face-detection-retail-0004`, to detect faces and predict their bounding boxes;
+* `landmarks-regression-retail-0009`, to predict face keypoints;
+* `face-reidentification-retail-0095`, to recognize readers.
+
+For more information about the pre-trained models, refer to the [model documentation](../../../models/intel/index.md).
+
+### How it works
+
+The application is started from the command line.
+It reads video stream frame-by-frame from a web-camera device and performs independent analysis
+of each frame. To make predictions the application uses 3 models. An input frame is processed by
+the face detection model to detect bounding boxes. Then face keypoints
+are predicted by the facial landmarks regression model. Keypoints are used
+to align the face and match it with faces in the database. 
+To register in the library press `r` on keyboard. After the reader has been registered, he will be authorized 
+through face recognition. Registration allows receiving and returning books by recognizing QR codes for 
+each book. To make book recognition press `b` on the keyboard. Also application provides some 
+extra statistics in console, like a list of registered readers, full list of books in the library,
+history of borrowing books. To change the information in console press `f` on the keyboard. To exit press `q`.
+
+### Installation and dependencies
+
+To install all the required Python modules you can use:\
+'''
+pip install -r requirements.txt 
+'''
+
+### Creating QR-codes for books
+
+The following files are used to create QR-codes for books:
+* `library.json` file contains information about books in the library. 
+* `createQRCodes.py` script generates QR-codes for each book in `library.json` file.
+
+``` sh
+usage: createQRCodes.py [-h] [-i LIB] [-o OUT]
+
+optional arguments:
+  -h, --help  show this help message and exit
+  -i LIB      unput `.json` file with info
+  -o OUT      directory to save generated QR-codes
+```
+
+### Running the demo
+
+Running the application with the `-h` option or without
+any arguments yield the following message:
+
+``` sh
+python ./smart_library_demo.py -h
+
+usage: smart_library_demo.py [-h] -fr FREC -m_rd RDMODEL -fd FDDET -m_fd
+                             FDMODEL -lm LMDET -m_lm LMMODEL [-w_rd RDWIDTH]
+                             [-h_rd RDHEIGHT] [-t_rd RDTHRESHOLD]
+                             [-w_fd FDWIDTH] [-h_fd FDHEIGHT]
+                             [-t_fd FDTHRESHOLD] [-w_lm LMWIDTH]
+                             [-h_lm LMHEIGHT] [-br BR] [-lib LIB] [-w WEB]
+                             [--no_show]
+
+Smart Library Sample
+
+optional arguments:
+  -h, --help         show this help message and exit
+  -fr FREC           Required. Type of recognizer. Available DNN face
+                     recognizer - DNNfr
+  -m_rd RDMODEL      Required. Path to .xml file
```

Comment:
Please specify which kind of model needs here.
Same for `-m_fd`

Reply:
Done in commit https://github.com/opencv/open_model_zoo/pull/882/commits/460d3bb33951bee775dae2fbc0eb7b8bf96214c7